terms/neuro/study.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "$id": "sage.annotations-neuro.study-0.0.25",
+    "$id": "sage.annotations-neuro.study-0.0.26",
```

Comment:
Since there was another PR for study names merged before this one, will you please merge master into this and re-update the version number to 0.0.27?

Reply:
Done! And I rejiggered my BBEdit settings so that it auto-detabs when I save, hopefully that will deal with the tabs/spaces issue in the futre.